base/grafana/README.md

```diff
@@ -2,9 +2,33 @@
 
 [Grafana](https://https://grafana.com/) is an open-source analytics dashboard application.
 
-A Grafana instance is part of the default Sourcegraph cluster installation. 
+A Grafana instance is part of the default Sourcegraph cluster installation.
 
 ## Namespaces
 
 If you are deploying Sourcegraph to a non-default namespace, you'll have to change the namespace specified in
  [grafana.ClusterRoleBinding.yaml](grafana.ClusterRoleBinding.yaml) to the one that you created.
+
+## Docker image
+
+We are running our own image of Grafana which contains a standard Grafana installation packaged together with provisioned dashboards.
+For details see [Grafana Docker Image](https://github.com/sourcegraph/sourcegraph/blob/master/docker-images/grafana/README.md)
+
+## Exposing Grafana directly
```

Comment:
These docs should go into https://docs.sourcegraph.com/admin/monitoring_and_tracing not here given admins will need them. Anything in this dir should be specific to devs

Reply:
i see, ok. i'll move